docs/CODING_STYLE.md

```diff
@@ -519,6 +519,21 @@ SPDX-License-Identifier: LGPL-2.1-or-later
           return log_error_errno(SYNTHETIC_ERRNO(EIO), "Failed to read ...");
   ```
 
+- If the error shall be ignored rather than propagated, insert ", ignoring"
+  at the end:
+
+  ```c
+  r = parse(...);
+  if (r < 0)
+          log_debug_errno(r, "Failed to parse ..., ignoring: %m");
+  else
+          parsed = r;
+  ```
+
+- Insert a period at the end of the log message if no errno shall be logged.
+  Bus errors however should not end with a period, because they are passed to
+  the client where the message could be embedded within other texts.
```

Comment:
I don't like this change. If people want to embed a provided error message, then they have responsible to insert spaces around, lower the fist character, drop the terminating period if necessary, and so on.
So, period at the end should not take any special treatment.

Reply:
I am not sure what should we do but considering that even dbus-broker doesn't adhere to the rule proposed here, I'd agree with @yuwata. For example,

https://github.com/bus1/dbus-broker/blob/main/src/launch/launcher.c#L1301

Reply:
I would prefer to drop this whole paragraph. We have periods in various places, but also quite a lot of code doesn't have it, and it's not really a bug if they are missing. In many cases the message is not really a sentence and the period is not needed. So I think it makes sense to leave this undocumented and use whatever makes sense in context.

Reply:
So I think it's a lot of work trying to clean this up, to the point that it's probably not worth it. 

Also, I think the messages associated with some of our bus errors use multiple sentences, and it would be really weird if they'd have an inner full stop, but not one at the end.

Dunno, I am not against making a recommendation regarding this, but it's really not clear to me what that recommendation would be and I kinda sense it would become quite complex to make it comprehensively useful.